networking/eth2/src/testFixtures/java/tech/pegasys/teku/networking/eth2/Eth2P2PNetworkFactory.java

```diff
@@ -351,6 +352,8 @@ protected Eth2P2PNetwork buildNetwork(final P2PConfig config) {
                                 config.getTargetSubnetSubscriberCount(),
                                 subnetPeerCountGauge),
                         reputationManager,
+                        new AtomicReference<>(),
+                        NodeIdToDataColumnSidecarSubnetsCalculator.NOOP,
```

Comment:
Is this NOOP correct here?

Reply:
It's test fixtures, don't need it at this point there, I think